File: NewProject/Models/SPGG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace NewProject.Models
{
    //药品规格表
    public class SPGG : Table
    {
        [Key]
        public int GUID { get; set; }
        [Display(Name = "规格编号")]
        public string GGBH { get; set; }
        [Display(Name = "商品编号")]
        public int SPBH { get; set; }
        [Display(Name = "药品规格")]
        public string GGMC { get; set; }
        [Display(Name = "包装数量")]
        public string BZSL { get; set; }
        [Display(Name = "包装单位")]
        public string BZDW { get; set; }
    }
}

File: NewProject/wwwroot/js/Class/Spgg.js
//商品规格
export default class Spgg {
    constructor() {
        //药品规格
        this.ggmc = "";
        //包装数量
        this.bzsj = "";
        //包装单位
        this.bzdw = "";
    }
    addData(obj) {
        this.ggmc = obj.ggmc;
        this.bzsj = obj.bzsj;
        this.bzdw = obj.bzdw;
    }
}